kernel/src/actions/deletion_vector.rs

```diff
@@ -358,4 +460,32 @@ mod tests {
         expected[4294967300] = false;
         assert_eq!(bools, expected);
     }
+
+    #[tokio::test]
+    async fn test_dv_wrapper() -> Result<(), Box<dyn std::error::Error>> {
+        let example = dv_inline();
+
+        let de_comp = z85::decode(&example.path_or_inline_dv)?;
+        dbg!(de_comp.len());
```

Comment:
aside: what's the difference between `debug!` and `dbg!`?

Reply:
It's basically debug print, for when I'm hacking things together, it'd be roughly the same as `println!("{:?}", some_value);`